roles/tests/test_api.py

```diff
@@ -76,6 +80,63 @@ def test_roles_init_new_repo_2(self):
             sorted(RepoPermission.author_permissions())
         )
 
+    # pylint: disable=invalid-name
+    def test_roles_init_new_repo_fake_permission_admin(self):
+        """
+        Non existing permissions for admin
+        """
+        with patch.object(api.RepoPermission,
+                          'administrator_permissions') as mock_method:
+            mock_method.return_value = ['fake_permission']
+            repo = create_repo(
+                name=self.repo_name,
+                description=self.repo_desc,
+                user_id=self.user.id,
+            )
+            admin = Group.objects.get(name=self.group_admin)
+            self.assertListEqual(
+                get_perms(admin, repo),
+                []
+            )
+
+    # pylint: disable=invalid-name
+    def test_roles_init_new_repo_fake_permission_curator(self):
+        """
+        Non existing permissions for curator
+        """
+        with patch.object(api.RepoPermission,
+                          'curator_permissions') as mock_method:
+            mock_method.return_value = ['fake_permission']
+            repo = create_repo(
+                name=self.repo_name,
+                description=self.repo_desc,
+                user_id=self.user.id,
+            )
+            curator = Group.objects.get(name=self.group_curator)
+            self.assertListEqual(
+                get_perms(curator, repo),
+                []
+            )
+
+    # pylint: disable=invalid-name
```

Comment:
Just add one of these right below the class declaration instead of repeating them.